README.md

```diff
@@ -1 +1,37 @@
-# Capstone-1
\ No newline at end of file
+# Capstone Project
+
+School of Community Leadership-First capstone project.
+
```

Comment:
Kindly add a screenshot of your website in your README file to make the README file more professional.

Reply:
Addressed